src/less/dropdown/ds6/dropdown-base.less

```diff
@@ -15,6 +15,18 @@ span.listbox {
 
 // BEM elements
 
+.menu button.expand-btn--borderless,
+.fake-menu button.expand-btn--borderless {
+    border: none;
```

Comment:
Lesshint is throwing a warning about border: none.

```
Warning: dropdown-base.less: line 20, col 13, borderZero: Border properties should use 0 instead of "none".
```

Reply:
Yeah, let me grab that one. Old habits die hard ...